cas-server-core-authentication/src/test/java/org/jasig/cas/authentication/FileTrustStoreSslSocketFactoryTests.java

```diff
@@ -18,7 +19,8 @@
 public class FileTrustStoreSslSocketFactoryTests {
 
     @Test
-     public void verifyTrustStoreLoadingSuccessfullyWithCertAvailable() throws Exception {
+    @Ignore
```

Comment:
Not according to Travis. Not sure why it fails locally, but Travis runs it fine, right?